Lesson7_OOPS.cpp

```diff
@@ -0,0 +1,106 @@
+/* DESIGN A VENDING MACHINE */
+#include<iostream>
+using namespace std;
+class Inventory
+{
+	int coke,sprite,chips;
+	static int coke_count,sprite_count,chips_count;
+	public:
+	Inventory()
```

Comment:
It would be nice if this constructor had some kind of map of item -> quantity so these values could be changed